dashboard/src/main/java/com/github/ladicek/rourka/jenkins/JenkinsClient.java

```diff
@@ -57,7 +53,16 @@ public void init() {
                 .filter(it -> it.description != null && !it.description.isEmpty())
                 .map(it -> {
                     String description = it.description.replaceAll("<!-- .*? -->", "");
-                    JsonJobDataInDescription data = gson.fromJson(description, JsonJobDataInDescription.class);
+
+                    JsonJobDataInDescription data;
+                    try {
+                        data = gson.fromJson(description, JsonJobDataInDescription.class);
+                    } catch (JsonSyntaxException exception){
+                        return new Job(null,null,null,null);
+                    }
+                    if (data == null){
+                        return new Job(null,null,null,null);
+                    }
```

Comment:
This is interesting. Agree we want to do something about this, not sure if this is the correct way.

Reply:
Fixed in https://github.com/rhoar-ci/dashboard/commit/425cd2dcdeb1f5a9829b2d935ce631d79e65104b